### PR TITLE
Tickets 12,15,16

### PR DESCRIPTION
**Pull request by adi-124**

### CODE REVIEW

File: src/test/java/steps/InsightManagementSteps.java
Comment (shyambi) [inline src/test/java/steps/InsightManagementSteps.java:1]: @adi-124 Let me know the reason of deletion here.

File: src/test/java/steps/InvestigationManagementSteps.java
Comment (shyambi) [inline src/test/java/steps/InvestigationManagementSteps.java:1]: @adi-124 Let me know the reason of deletion here.

File: src/test/java/steps/ManageUserActivitySteps.java
Comment (shyambi) [inline src/test/java/steps/ManageUserActivitySteps.java:1]: @adi-124 Let me know the reason of deletion here.

File: src/test/java/steps/MessageFlowDataQualitySteps.java
Comment (shyambi) [inline src/test/java/steps/MessageFlowDataQualitySteps.java:1]: @adi-124 Let me know the reason of deletion here.

File: src/test/java/steps/MessageFlowELRReportSteps.java
Comment (shyambi) [inline src/test/java/steps/MessageFlowELRReportSteps.java:1]: @adi-124 Let me know the reason of deletion here.

File: src/test/java/steps/FavoriteIntaractChartsteps.java
Comment (shyambi) [inline src/test/java/steps/FavoriteIntaractChartsteps.java:1]: @adi-124 Let me know the reason of deletion here.

**Review by shyambi — COMMENTED** [review]
@adi-124 Please check my comments. You have delete some step files. Please add the reason.

**Comment by adi-124** [comment]
@shyambi i have converted all these classes into one class using scenario outlines. So that way i have deleted all of these

**Comment by adi-124** [comment]
resolve conflict please accept PR @sarker02

File: src/test/resources/features/InsightAynalytic.feature
Comment (shyambi) [inline src/test/resources/features/InsightAynalytic.feature:1]: Please use old name. InsightAnalytic.feature

File: src/test/resources/features/Management.feature
Comment (shyambi) [inline src/test/resources/features/Management.feature:1]: Please add reason of deleting this file here.

File: src/test/resources/features/MessageFlow.feature
Comment (shyambi) [inline src/test/resources/features/MessageFlow.feature:1]: Please add reason of deleting this file here.

File: src/test/resources/features/SyndromicAnalytic.feature
Comment (shyambi) [inline src/test/resources/features/SyndromicAnalytic.feature:1]: Please add reason of deleting this file here.

File: src/test/resources/features/InsightAnalytic.feature
Comment (shyambi) [inline src/test/resources/features/InsightAnalytic.feature:1]: Please add reason of deleting this file here.

File: src/test/java/steps/UtilizationDashboardSteps.java
Comment (shyambi) [inline src/test/java/steps/UtilizationDashboardSteps.java:1]: Please add reason of deleting this file here.

File: src/test/java/steps/ManagementInvestigationResolutionSteps.java
Comment (shyambi) [inline src/test/java/steps/ManagementInvestigationResolutionSteps.java:1]: Please add reason of deleting this file here.

File: src/test/java/pages/FavoritesIntractChartPage.java
Comment (shyambi) [inline src/test/java/pages/FavoritesIntractChartPage.java:1]: Please add reason of deleting this file here.

File: src/test/java/pages/ForgotPasswordPage.java
Comment (shyambi) [inline src/test/java/pages/ForgotPasswordPage.java:1]: Please add reason of deleting this file here.

File: src/test/java/pages/EssenceAlertPage.java
Comment (shyambi) [inline src/test/java/pages/EssenceAlertPage.java:1]: Please add reason of deleting this file here.

File: src/test/java/pages/FavoritePage.java
Comment (shyambi) [inline src/test/java/pages/FavoritePage.java:1]: Please add reason of deleting this file here.

File: config/global.properties [hunk 1]
```diff
@@ -5,9 +5,10 @@ site_url=https://solutions-qa.inductivehealth.com/home
 browser=chrome
 
 ## Project root directory's absolute path
-project_root_dir=C:/Users/www.abcom.in/eclipse-workspace/HealthQA
-
+project_root_dir=C:\Users\Abrar\eclipse-workspace\New folder\New folder\HealthQA
 
 ## Firefox and chrome drivers path
 firefox_driver_path=C:/drivers/geckodriver.exe
+chrome_driver_path=C:/drivers/geckodriver.exe
```
Comment (sarker02) [inline config/global.properties:12]: Change the Path back to firefox_driver_path

Comment (adi-124) [reply config/global.properties:12]: we are using Web driver manager so there is no need driver path.

**Pull request merged**